src/styles/watch.scss

```diff
@@ -0,0 +1,35 @@
+.watch {
+  @include circle (500px);
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+  background-color: rgb(165, 215, 224);
```

Comment:
you use `rgb`, `hex` and `color-names` in this task, you should stick with one of them, i suggest you to stick with `hex` values